ps_specials.php

```diff
@@ -227,15 +226,27 @@ private function getSpecialProducts()
 
         $presenterFactory = new ProductPresenterFactory($this->context);
         $presentationSettings = $presenterFactory->getPresentationSettings();
-        $presenter = new ProductListingPresenter(
-            new ImageRetriever(
-                $this->context->link
-            ),
-            $this->context->link,
-            new PriceFormatter(),
-            new ProductColorsRetriever(),
-            $this->context->getTranslator()
-        );
+        if (version_compare(_PS_VERSION_, '1.7.4', '>=')) {
```

Comment:
```suggestion
        if (version_compare(_PS_VERSION_, '1.7.5', '>=')) {
```

1.7.4 is still using the old path, change the condition and it should work ;-)